wcag20/sources/understanding/navigation-mechanisms-link.xml

```diff
@@ -6,19 +6,19 @@
    <div3 role="intent">
       <head/>
 
-<p>この達成基準の意図は、利用者がコンテンツ内にある各リンクの目的を理解するのを手助けすることで、彼らはそれを辿ることを望むかどうかを、彼らが決めることができる（同じ機能を持っているコンポーネントの識別に一貫性を求めるための<loc href="consistent-behavior-consistent-functionality" linktype="guideline">達成基準 3.2.4</loc> も見よ）。リンクの目的は、そのリンク テキストから識別されることができるので、ユーザエージェントが、ページ上にある全てのリンクの一覧を提供するときのように、リンクは、それらが文脈の中にあるとき、理解することができる。</p>
-<p>リンク内のテキストは、リンクの目的を説明するように意図されている。リンクが、文書又はウェブ アプリケーションの一つへ導く場合は、文書またはウェブ アプリケーションの名前は、（文書またはウェブ アプリケーションにあなたを導く）リンクの目的を説明するのに十分であろう。文書又はウェブ アプリケーションの名前を使用する必要がないことに注意されたい。他のものが、リンクの目的を説明することもできる。</p>
-<p> <loc href="navigation-mechanisms-title" linktype="understanding">達成基準 2.4.2</loc> は、ページのタイトルに取り組んでいる。ここでもまた、ページに表示されている文書とウェブ アプリケーションの名前は、ページの目的を説明するのに十分であろう。リンクとタイトルが一致する又はとても似ていることは、良い習慣であり、リンクの「クリック」と利用者が着地するウェブ ページ間の継続性を提供する。</p>
+<p>この達成基準の意図は、利用者がコンテンツ内にある各リンクの目的を理解するのを手助けすることで、利用者がそのリンクを辿ることを望むかどうかを判断できるようにすることである。同じ宛先をもつリンクは同じ記述があるが、目的及び宛先が異なるリンクは異なる記述があるのがベストプラクティスである (同じ機能を持つコンポーネントの識別に一貫性を求めている<loc href="consistent-behavior-consistent-functionality" linktype="guideline">達成基準 3.2.4</loc> も併せて参照のこと)。リンクの目的はリンクテキストから識別できるので、ユーザエージェントがページ上にある全てのリンクの一覧を提供する場合など、リンクが文脈の中にあるときに、リンクを理解できる。</p>
+<p>リンク内のテキストは、リンクの目的を説明するように意図されている。リンクが利用者を文書又はウェブアプリケーションへ導く場合、その文書又はウェブアプリケーションの名前は、リンクの目的 (その文書又はウェブアプリケーションに導くためのもの) を説明するものとして十分であろう。文書又はウェブアプリケーションの名前を使用するよう要求されないことに注意すること。他のものも、リンクの目的を説明することがある。</p>
+<p><loc href="navigation-mechanisms-title" linktype="understanding">達成基準 2.4.2</loc> は、ページのタイトルに取り組んでいる。ここでもまた、ページに表示されている文書又はウェブアプリケーションの名前は、ページの目的を説明するのに十分であろう。リンクとタイトルが一致する又は極めて似ていることは、グッドプラクティスであり、リンクの「クリック」と利用者が着地するウェブページとの間の継続性を提供する。</p>
 <p>この達成基準には、リンクの目的をウェブページ上にある情報からは判断できないリンクに対して例外が認められている。そのような状況では、障害のある利用者が不利な立場にあるわけではない。リンクの目的を理解するために追加入手可能な文脈が一切ないのである。しかし、この達成基準を満たすためには、そのリンクの目的を解釈するためにそのウェブページ上で利用できる文脈はどんなものであれ、そのリンクテキストで入手可能にしなればならない。 </p>
-<p>「メカニズム」という用語を用いているのは、コンテンツ制作者がすべてのリンクを前後の文脈に関係なく完全に理解できるようにするか、あるいはそのようにする方法を提供できるようにするためである。リンクすべてをリンクテキストだけでも曖昧ではないようにすることで、ページによっては、分かりやすくなる利用者もいれば、分かりづらくなってしまう利用者もいる。リンクを（それ自身により）明らかにするか そうしないか できるようにすることは、障害のある利用者にも無い利用者にも、彼らのニーズに最も合致したフォーマットのページを使えるようにすることである。</p>
-<p>例えば、100 冊の書籍のタイトルが並んでいるページに、それぞれの書籍を HTML、PDF、DOC、TXT、MP3、又は AAC でダウンロードするリンクがある場合、たいていは書籍のタイトルの後に「HTML 版」のようなリンクがある。そして、「次のバージョンも利用可能:」という文が続いていて、「HTML」、「PDF」、「DOC」、「TXT」、「MP3」、そして「AAC」という短いテキストのリンクがある。レベル AAA では、ある利用者は このようにページを見ることが選べるであろう。それぞれのリンクに書籍のタイトルがあると、分かりづらかったり、使うのに時間がかかってしまったりするからである。その他の利用者は、それぞれのリンクがそれだけで理解できるように、それぞれのリンクの一部に書籍のタイトルがあるようにページを見ることが選べるであろう。前者のグループにも後者のグループにも、いろいろな使い方をしている、又は様々な種類や度合いの障害のある、視覚又は認知の障害のある利用者が含まれるであろう。</p>
+<p>「メカニズム」という用語を用いているのは、コンテンツ制作者がすべてのリンクを前後の文脈に関係なく完全に理解できるようにするか、あるいはそのようにする方法を提供できるようにするためである。リンクすべてをリンクテキストだけでも曖昧ではないようにすることで、ページによっては、分かりやすくなる利用者もいれば、分かりづらくなってしまう利用者もいる。リンク（それ自体を）曖昧さなく明示するようにもしないようにもできることで、障害のある利用者にもない利用者にも、彼らのニーズに最も合致したフォーマットでページを使えるようにすることができる。</p>
```

Comment:
カッコ内を取っても日本語として違和感がなく繋がる文章にしたいです。

>Providing the ability to make the links unambiguous (by them selves) or not provides both users with disabilities with the ability to use the page in the format that best meets their needs.

→リンクが (それ自体) 曖昧でなくなるようにしたり、そのままにしたりできるようにすることで、障害のある利用者、ない利用者の双方が、彼らのニーズに最も合致したフォーマットでページを利用できるようになる。

とか?
実は前の段落では themselves を勝手に「リンクテキスト」と補って訳しているのでそれに揃えてもいいのですが、どちらかというと前の段落を直した方が原文に忠実にはなりますね。

>This is done because for some pages, making the links all unambiguous by themselves makes the pages easier for some users and harder for others. 

→こうしたのは、ページによっては、全てのリンクをそれ自体で曖昧さがなくなるようにすると、ある利用者に分かりやすくなると同時に、他の利用者には分かりづらくなってしまうからである。


あと最初の段落の either がちゃんと訳されていないかもしれません。
いずれか選択できることに意味がある、というのがこの段落全体の話なので、もうちょっと頑張って訳出した方がいいかも。

>The word "mechanism" is used to allow authors to either make all links fully understandable out of context by default or to provide a way to make them this way. 

→ここで「メカニズム」という言葉が使われているのは、コンテンツ制作者に対して、デフォルトで全てのリンクをコンテキストなしに完全に理解できるようにするか、あるいはそのようにするための手段を提供するかのいずれかを選べるようにするためである。

※「選べるようにする」はかなり意訳かも